### DISCUSSION TITLE
I don't want to show Dependency-Track Project on Jenkins

### DISCUSSION THREAD

**Discussion by cl-emanuele-ferrara** [DependencyTrack/dependency-track#2968] [Q&A]
I have integrated dependency track on **Jenkins** but I want to show only the vulnerability report (**Dependency-Track Report**) in Jenkins pipeline, I don't want to show the link to the project in the application (**Dependency-Track Project**)
![image](https://github.com/DependencyTrack/dependency-track/assets/103640930/10d6413e-8782-416f-9909-0b4923a5b244)

I don't want to show, therefore, not even the writing "These results may not be up to date. Please visit the Dependency-Track project page for most recent results" 
![image](https://github.com/DependencyTrack/dependency-track/assets/103640930/1f4c96dc-2809-4ca1-a53c-4c32fc89135a)

Is it possible to do it?
I tried changing the permissions in the Teams section, but if I try to remove "View_Vulnerability/View_Portfolio" the Dependency-Track Report is also removed (but I want this).

**Comment by syalioune** [comment]
Hello @cl-emanuele-ferrara 
You should ask your question in Dependency Track jenkins plugin repository https://github.com/jenkinsci/dependency-track-plugin
Folks there would have a better knwoledge of the plugin inner working.
From what I can see after a quick source code scan, it seems to not be possible to remove that warning header.
